config/app.php

```diff
@@ -1,12 +1,15 @@
 <?php
 
-use Cake\Cache\Engine\FileEngine;
+use Cake\Cache\Engine\FileEngine as CacheEngine;
 use Cake\Database\Connection;
-use Cake\Database\Driver\Mysql;
+use Cake\Database\Driver\Mysql as DatabaseDriver;
 use Cake\Error\ExceptionRenderer;
-use Cake\Log\Engine\FileLog;
+use Cake\Log\Engine\FileLog as LogEngine;
 use Cake\Mailer\Transport\MailTransport;
 
+$appName = 'myapp';
+$cachePrefix = "${appName}_";
```

Comment:
I like this consolidation, I'm less sure about the class aliasing being simpler for folks new to CakePHP.